source/indexes.txt

```diff
@@ -0,0 +1,274 @@
+.. _php-indexes:
+
+=================================
+Optimize Queries by Using Indexes
+=================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :description: Learn how to use indexes by using the MongoDB PHP Library.
+   :keywords: query, optimization, efficiency, usage example, code example
+
+.. .. toctree::
+..    :titlesonly:
+..    :maxdepth: 1
+.. 
+..    /work-with-indexes
+
+Overview
+--------
+
+On this page, you can see copyable code examples that show how to manage
+different types of indexes by using the {+php-library+}.
+
+.. .. tip::
+.. 
+..    To learn more about working with indexes, see the :ref:`php-work-indexes`
+..    guide. To learn more about any of the indexes shown on this page, see the link
+..    provided in each section.
+
+To use an example from this page, copy the code example into the
+:ref:`sample application <php-index-sample>` or your own application.
+Be sure to replace all placeholders in the code examples, such as
+``<connection string>``, with the relevant values for your MongoDB
+deployment.
+
+.. _php-index-sample:
+
+.. include:: /includes/usage-examples/sample-app-intro.rst
+
+.. literalinclude:: /includes/usage-examples/sample-app.php
+   :language: php
+   :copyable:
+   :linenos:
+   :emphasize-lines: 10-12
+
+Single Field Index
+------------------
+
+The following example creates an ascending index on the specified field:
```

Comment:
I noticed that this is the only example introduction that specifies ascending index – could be a good idea to ensure that it's called out for all other examples that create an ascending index.